.gitignore

```diff
@@ -1,2 +1,2 @@
-/target
+target

```

Comment:
you've changed this file multiple times now. I think you should just keep the most recent version of the file